backend/lib/endpoints/sendgrid.js

```diff
@@ -0,0 +1,48 @@
+const axios = require("axios");
+const { sendgridContactSchema } = require("./schema/sendgrid");
+const {
+  config: { sendgrid: sendgridConfig },
+} = require("../../config");
+
+const populateSendgridContactsObj = (body) => {
+  return {
+    contacts: [
+      {
+        email: body.email,
+        ...body.location,
```

Comment:
doesn't this have to be assigned to `location` property instead of spreading all the props?

Reply:
No, it actually needs to be spreaded. There is no location field on sendgrid, so instead I am spreading this object, which contains city, state, and country, and those exist on sendgrid, so they get applied.